icicle/src/fields/field.cu

```diff
@@ -1,6 +1,11 @@
+#ifdef CURVE_ID
```

Comment:
maybe duplicate the file and avoid the coupling? it's so small anyway and not sharing much